src/entities/removeLiquidityBoostedV3/doRemoveLiquidityProportionalQuery.ts

```diff
@@ -0,0 +1,33 @@
+import { createPublicClient, http, getAddress } from 'viem';
+
+import { BALANCER_COMPOSITE_LIQUIDITY_ROUTER, CHAINS } from '@/utils';
+
+import { Address } from '@/types';
+
+import { balancerCompositeLiquidityRouterAbi } from '@/abi';
+
+import { RemoveLiquidityProportionalInput } from 'src';
+
+export const doRemoveLiquidityProportionalQuery = async (
+    { rpcUrl, chainId, bptIn }: RemoveLiquidityProportionalInput,
```

Comment:
similar to adds, can update to accept address and userData

Reply:
Addressed